.ci/windows-steps.yaml

```diff
@@ -62,6 +64,7 @@ steps:
     -DBOOST_INCLUDEDIR=$(Agent.ToolsDirectory)\boost.1.60.0.0\lib\native\include `
```

Comment:
What happens if you change `\include` to `\tmp\include` for `ARMADILLO_INCLUDE_DIR`?

Reply:
Or maybe you can just leave `ARMADILLO_LIBRARY` out?